Remotion/Web/Core/UI/Controls/WebTreeView.cs

```diff
@@ -772,7 +772,7 @@ private void RenderNodes (HtmlTextWriter writer, WebTreeNodeCollection webTreeNo
           writer.AddAttribute(DiagnosticMetadataAttributes.TriggersPostBack, "true");
       }
       writer.AddAttribute("tabindex", "-1");
-      writer.AddAttribute(HtmlTextWriterAttribute2.Role, HtmlRoleAttributeValue.None);
+      writer.AddAttribute(HtmlTextWriterAttribute2.AriaHidden, HtmlAriaHiddenAttributeValue.True);
```

Comment:
Ah, yes. The assumtion was that role=none would be sufficient. for some reason, it wasn't. Or, actually, the problem is that axe core interprets role=none as still aria-visible. either way, aria-hidden takes care of it.